ocs_ci/ocs/cluster.py

```diff
@@ -2242,7 +2248,11 @@ def __init__(self, fstrim=False, fail_on_thin_pool_not_empty=False):
             ]
         func_list.extend(extend_func_list)
 
-        thread_init_class(func_list, shutdown=0)
+        if self.thread_init:
```

Comment:
For the theory, if self.thread_init is False, where doe's the class init?
Anyway I can think of a scenario that I want longer initiation times for the class so please remove this, unless you have some good explanation why to init the class in serial.